ci/tests/puppeteer/run.sh

```diff
@@ -97,6 +102,14 @@ while (( "$#" )); do
     BUILDFLAGS="${BUILDFLAGS} --offline"
     shift 1;
     ;;
+  --max-workers|--mw)
+    MAX_WORKERS="$2"
+    shift 2
+    ;;
+  --dep-cache-days|--dcd)
+    BUILDFLAGS="${BUILDFLAGS} -PdependencyCacheDays=$2"
```

Comment:
I dont think this needs to be configurable. I'd just fix the value to be at 10 days, etc and remove the option. This will never be used by anyone, and it's fairly obsecure.